.github/ci-bionic/after_make.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -l
+
+set -x
+
```

Comment:
do we want to add `set -e` to be sure of existing on errors?

Reply:
Done 
f8755ff